doc/release/upcoming_changes/25388.new_feature.rst

```diff
@@ -0,0 +1,9 @@
+`numpy.linalg.cholesky` and `numpy.linalg.pinv` new parameters
+--------------------------------------------------------------
+
+The ``upper`` and ``rtol`` keyword parameters were added to
+`numpy.linalg.cholesky` and `numpy.lingalg.pinv`, respectively, to
+improve array API compatibility.
+
+For `numpy.linalg.pinv` if neither ``rcond`` nor ``rtol`` is specified,
+the ``rcond``'s default is used.
```

Comment:
Maybe worth stating that eventually we hope to deprecate `rcond`?

Reply:
Sure! Added.